### ISSUE TITLE
I've run through the code, poor result

### ISSUE THREAD

**Issue by Greenleaf88** [piiswrong/deep3d#26]
Although I mansged to run through the test code with my own dataset, the result is very poor, the right eye images are rather blur compared with left one. Could you please share the file mean.npz ?  My test data set is very small and may be the mean values generated from it are not very good?

**Comment by dongcin** [comment]
@Greenleaf88 I also want to run the code ,but I meet the cannot find Cudnn BatchNorm.Can you help me solve the problem? can you tell me the vesion of your software ? Thanks a lot

**Comment by Greenleaf88** [comment]
@dongcin I've also met with lots of version problems, using CUDA 7/CUDNN 4 would be OK.

**Comment by dongcin** [comment]
@Greenleaf88  Thanks for your reply. I download the cuda 7.0 But I find that the driver version is 7.5 running version is 7.0 .Have you met the problem

**Comment by AjayNandoriya** [comment]
cuda version is not important..CUDNN v4 is important. I am able to get good result with cuda 8.

**Comment by phonchi** [comment]
@Greenleaf88 @AjayNandoriya  also want to test the code. However, the download link of  [deep3d-0050.params](http://homes.cs.washington.edu/~jxie/download/deep3d-0050.params) seems dead now. Do you have backup version of this file?  Thanks in advance!!

**Comment by Samo-777** [comment]
with cuda 7.5 and cudnn4.

Completely  shiiiiiiiiiiiiiiit results with demo images.  

If you can get results like the demo, you should sell it to film makers to be a billionare

**Comment by vinthony** [comment]
@phonchi Do you find the pre-trained model?

**Comment by liutianling** [comment]
@Greenleaf88 Hi。I'm learning the project too. Can you tell me hao to run the code. Or how to make my own lmdb database.Thanks!

**Comment by liutianling** [comment]
@Samo-777 Can you tell me hao to run the convert_movie.py ?
I first use parse.py and data.py to get the lmdb database. But meet problems in data.py. I don't know hao to give the parameter of the prefix in parse.py and the parameter of source in convert_movie.py.
Thanks a lot!

**Comment by Merofine** [comment]
@liutianling how to run data.py to get the lmdb database .
>>python data.py
usage: data.py [-h] path fdb
data.py: error: too few arguments


i do not how to run it.please help me !Thanks a lot

**Comment by Tord-Zhang** [comment]
@Samo-777 Haha, bro, that's really mean. But I agree with you, because I have just run the demo code.......
